task09-reactjs/task09/src/components/Header.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary js for the header component
+ */
+
+import '../scss/Body.scss';
+function Header() {
+  return (
+    
+      <header>
+        <nav className={`navbar fixed-top navbar-expand-xl navbar-light bg-white py-4 justify-content-center`}>
+          <button className='navbar-toggler burger-menu' type='button' data-toggle="collapse" data-target="#navbarTogglerDemo01"
+          aria-controls="navbarTogglerDemo01" aria-expanded="false" aria-label="Toggle navigation">
+            <span id="burger-icon" className="icon-align-left"></span>
+            <span id="burger-icon" className="icon-times"></span>
+          </button>
+          <div className="logo">
+            <img className="one-piece-logo" src={require("../images/one-piece-logo.png")} alt="One Piece Logo" />
+          </div>
+          <div className="collapse navbar-collapse" id="navbarTogglerDemo01">
+              <ul className="navbar-nav">
+                <li className="nav-item">
+                  <a className="nav-link active" href="/">Home<span class="sr-only">(current)</span></a>
```

Comment:
Avoid class in React please check all project and check this https://bobbyhadz.com/blog/react-invalid-dom-property-class 
<img width="828" alt="image" src="https://user-images.githubusercontent.com/48458005/186728814-4fc3c7e7-97af-4cab-b8c3-ab3704eec35a.png">